lib/api/middleware/set-model-form.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const setLinzNamespace = require('./set-linz-namespace');
+
+/**
+ * Set req.linz.model.form.
+ * @returns {Void} Calls the next middleware.
+ */
+const setModelForm = () => (req, res, next) => {
```

Comment:
Why does this need to be a function that returns a function? There are no arguments, so it would seem unnecessary?